db/new_seeds/scenarios/schools/fip_school.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module NewSeeds
+  module Scenarios
+    module Schools
+      class FipSchool
```

Comment:
As we're creating the school, partnership, cohort and programme here should the name be a bit more descriptive?

Reply:
Yes, struggling to think of a good name, it's a valid/populated/complete FIP school but without participants (or a SIT actually, perhaps that should be added).... anything spring to mind? `FipSchoolWithProvider` or `FipSchoolWithProgramme`?  I'm not sure they'd make sense to say it was a FIP School without those things though... 🤔

Reply:
or maybe the other bits could be added by chain methods? e.g. `FipSchool.with_partnership.with_programme.with_sit` ?

Reply:
This sounds good to me, so long as the additions aren't interdependent (I don't think they are) and can be added in any order/combination.

Reply:
Yes working on generalising this into just `School` with chainable options